fraim/cli.py

```diff
@@ -179,7 +180,7 @@ def workflow_options_to_cli_args(options_class: type[Any]) -> dict[str, dict[str
             arg_config["type"] = float
         elif get_origin(actual_type) is list:
             arg_config["nargs"] = "+"
-        elif get_origin(actual_type) is Union:
+        elif get_origin(actual_type) is Union or get_origin(actual_type) is UnionType:
```

Comment:
played it safe and added an or here, but feels like we can probably get rid of the first condition all together